otter_welcome_buddy/database/models/guild_model.py

```diff
@@ -4,6 +4,9 @@
 from otter_welcome_buddy.database.dbconn import BaseModel
 
 
+_GUILD_MODEL_TABLE_NAME = "guild"
```

Comment:
Why this var is on global scope? should it be in class with Final type?

Reply:
Oh, I wasn't aware of that, thanks for the suggestion I like more that approach!

Reply:
In fact, I think I can just use the same `__tablename__` that provide the `BaseModel`